storm-core/src/ui/public/templates/topology-page-template.html

```diff
@@ -131,33 +131,37 @@
     </tbody>
   </table>
 </script>
+
+<script id="topology-visualization-container-template" type="text/html">
```

Comment:
What is this change for? It looks like it just moved from topology-visualization-template to a new topology-visualization-container-template, that is now loaded through AJAX.